src/Host.h

```diff
@@ -619,7 +623,8 @@ class Host : public QObject
     bool mBubbleMode;
     bool mMapViewOnly = true;
     bool mShowRoomID;
-    bool mShowPanel;
+    bool mShowPanel; // Mapper
+    bool mShowVideoPlayerPanel;
```

Comment:
As per previous comment - I am assuming this refers to a showing/hiding a "control panel" with knobs on to work the video player and not the whole player widget...
```suggestion
    bool mShowVideoPlayerPanel = true;
```